docs/docs/illixr_plugins.md

```diff
@@ -2,290 +2,641 @@
+  - *Publishes* [`binocular_cam_type`][63] to `cam` topic.
+- `offload_vio.server_tx`
+  - Asynchronously *reads* [`imu_integrator_input`][66] from `imu_integrator_input` topic.
+  - *Publishes* a string to `vio_pose` topic.
+
+&nbsp;&nbsp;[**Details**][D17]&nbsp;&nbsp;&nbsp;&nbsp;[**Code**][C17]
+
+## openni
+
+Enables an interface to the Openni algorithms.
+
+Topic details:
+
+-   *Publishes* [`rgb_depth_type`][62] to `rgb_depth` topic. 
+
+&nbsp;&nbsp;[**Details**][D18]&nbsp;&nbsp;&nbsp;&nbsp;[**Code**][C18]
+
+## open_vins
+
+An alternate [_SLAM_][39] ([upstream][18]) implementation that uses a MSCKF
+(Multi-State Constrained Kalman Filter) to determine poses via camera/[_IMU_][36].
+
+Topic details:
+
+-   *Publishes* [`pose_type`][61] on `slow_pose` topic.
+-   *Publishes* [`imu_integrator_input`][66] on `imu_integrator_input` topic.
+-   Synchronously *reads*/*subscribes* to [`imu_type`][64] on `imu` topic.
+
+&nbsp;&nbsp;[**Details**][D19]&nbsp;&nbsp;&nbsp;&nbsp;[**Code**][C19]
+
+## openwarp_vk
+
+Provides a Vulkan-based reprojection service.
+
+Topic details:
+
+-   *Calls* [`vulkan::timewarp`][30]
+-   *Calls* [`pose_prediction`][32]
+
+&nbsp;&nbsp;[**Details**][D20]&nbsp;&nbsp;&nbsp;&nbsp;[**Code**][C20]
+
+## orb_slam3
+
+Utilizes the ORB_SLAM3 library to enable real-time head tracking (VIO).
+
+Topic details:
+
+-   Asynchronously *reads* [`binocular_cam_type`][63] from `cam` topic.
+-   *Publishes* [`pose_type`][61] to `slow_pose` topic.
+-   *Publishes* [`imu_integrator_input`][66] to `imu_integrator_input` topic.
+
+&nbsp;&nbsp;[**Details**][D21]&nbsp;&nbsp;&nbsp;&nbsp;[**Code**][C21]
+
+## passthrough_integrator
+
+Provides IMU integration.
+
+Topic details:
+
+-   Asynchronously *reads* [`imu_integrator_input`][66] from `imu_integrator_input` topic.
+-   Synchronously *reads* [`imu_type`][64] from `imu` topic.
+-   *Publishes* [`imu_raw_type`][65] to `imu_raw` topic.
+
+&nbsp;&nbsp;**Details**&nbsp;&nbsp;&nbsp;&nbsp;[**Code**][C22]
+
+## realsense
+
+Reads images and [_IMU_][36] measurements from the [Intel Realsense][25].
+
+Topic details:
+
+-   *Publishes* [`imu_type`][64] to `imu` topic.
+-   *Publishes* [`binocular_cam_type`][63] to `cam` topic.
+-   *Publishes* [`rgb_depth_type`][62] to `rgb_depth` topic.
+
+&nbsp;&nbsp;**Details**&nbsp;&nbsp;&nbsp;&nbsp;[**Code**][C23]
+
+## record_imu_cam
+
+Writes [`imu_type`][64] and [`binocular_cam_type`][63] data to disk.
+
+Topic details:
+
+-   Asynchronously *reads* [`binocular_cam_type`][63] from `cam` topic.
+-   Synchronously *reads* [`imu_type`][64] from `imu` topic.
+
+&nbsp;&nbsp;[**Details**][D24]&nbsp;&nbsp;&nbsp;&nbsp;[**Code**][C24]
+
+## record_rgb_depth
+
+Writes [`rgb_depth_type`][62] data to disk.
+
+Topic details:
+
+-   Synchronously *reads* [`rgb_depth_type`][62] from `rgb_depth` topic.
+
+**Details** [**Code**][C25]
+
+## rk4_integrator
+
+Integrates over all [_IMU_][36] samples since the last published [_SLAM_][39] [_pose_][37] to
```

Comment:
Let's not say "SLAM". Let's say visual-inertial poses. Same changes for gtsam_integrator above.

Reply:
fixed